src/components/BubbleModal.js

```diff
@@ -64,7 +70,14 @@ export const BubbleModal = ({ onClose, seedlingData, userId, seedlingKey }) => {
         <h2>{title}</h2>
         <div className={styles.linkPreviewContainer}>
           <div className={styles.linkPreview}>
-            {/* Link preview image goes here */}
+            {imgSrc && (
+              <Image
+                fill
+                src={imgSrc}
+                alt="Link preview"
+                className={styles.linkPreviewImage}
+              />
```

Comment:
With Next, you're not really supposed to use native `img` tags, because Next provides a lot of extra good stuff in their `Image` component.